extensions/panache/panache-mock/src/main/java/io/quarkus/panache/mock/PanacheMock.java

```diff
@@ -40,6 +41,13 @@ public static void mock(Class<?>... classes) {
         IsMockEnabled = !mocks.isEmpty();
     }
 
+    public static void mock(MockSettings settings, Class<?>... classes) {
+        for (Class<?> klass : classes) {
+            mocks.computeIfAbsent(klass, v -> Mockito.mock(klass, settings));
```

Comment:
What if the `settings` don't match from a previously cached mock you might have created for the same klass?

Reply:
I guess it will override the mock, since they're keyed on class and not settings. That's fine.

Reply:
and settings will not change?

Reply:
Ah, you're right, OK.

Reply:
Pfff, I'll just remove this method.